webapp/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
 
   app:
     # Go実装の場合は golang/ PHP実装の場合は php/
-    build: ruby/
+    build: php/
```

Comment:
Please do not change this line.

Reply:
Sorry, I fixed it.
Please review this again. @catatsuy